src/bookshelf.js

```diff
@@ -24,6 +24,9 @@ import Errors from './errors';
  * @param {Knex} knex Knex instance.
  */
 function Bookshelf(knex) {
+  if (!knex || !knex.__knex__) {
```

Comment:
This started throwing a warning when running tests:

    Knex:warning - knex.__knex__ is deprecated, you can get the module versionby running require('knex/package').version

Can you look into it?

Reply:
yep.  sorry i missed that.